src/Etterna/Models/StepsAndStyles/Steps.cpp

```diff
@@ -421,7 +425,8 @@ Steps::BorpNDorf(int modType)
 	dumbthings.clear();
 
 	if (modType < CalcPatternMod::ModCount && modType >= 0)
-		MinaSDCalcDebug(cereal, GAMESTATE->m_SongOptions.GetSong().m_fMusicRate,
```

Comment:
I'm not 100% clear on what this call does but should this not have a solo equivalent? I could be missunderstanding something

Reply:
This was a formatting change. There needs to be a Solo equivalent.